src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -99,6 +99,25 @@ if (RAJA_DIR)
         message(STATUS "RAJA loaded: ${RAJA_DIR}")
         set(RAJA_FOUND TRUE CACHE BOOL "")
     endif()
+
+    # Suppress warnings from cub and cuda related to the (low) version 
+    # of clang that XL compiler pretends to be.
+    if(C_COMPILER_FAMILY_IS_XL)
+        if(TARGET RAJA::cub)
+            blt_add_target_definitions(
+                TO RAJA::cub
+                SCOPE INTERFACE
+                TARGET_DEFINITIONS CUB_IGNORE_DEPRECATED_CPP_DIALECT)
+        endif()
+
+        if(TARGET cuda)
+            blt_add_target_definitions(
+                TO cuda
+                SCOPE INTERFACE
+                TARGET_DEFINITIONS THRUST_IGNORE_DEPRECATED_CPP_DIALECT)
+        endif()
+    endif()
```

Comment:
These avoid some annoying repeated warnings from cub and thrust about the Clang version needing to be greater than 6. They're not particularly useful to us when using the XL compiler, since it's C++ implementation is sufficiently modern.